Resources/public/css/views/dashboard/dashboard.css

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-dashboardblockbasedview,
+.ez-view-dashboardblockbasedview * {
+    box-sizing: border-box;
+}
+
+.ez-view-dashboardblockbasedview,
+.ez-view-dashboardblockbasedview > .pure-u {
```

Comment:
we try to get rid of PureCSS and for now this is clearly not needed so you should remove the selector with `pure-u` and the corresponding class in the template.